lib/builder.js

```diff
@@ -273,6 +273,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   var context = Object.create(global);
   context.System = configLoader;
   context.global = context.GLOBAL = context.root = context;
+  context.require = require;
```

Comment:
Strictly speaking, this should probably be a `require` scoped to the config file path itself and not the SystemJS Builder path as the local require here would be.

I think this can be done something along the lines of - `var Module = require('module'); var m = new Module(configPath); var require = m.require.bind(m)`?